definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -1315,13 +1308,6 @@
     "UserRole": "caseworker-divorce-courtadmin",
     "CRUD": "CRU"
   },
-  {
-    "LiveFrom": "01/01/2017",
-    "CaseTypeID": "DIVORCE",
-    "CaseEventID": "AddNote",
-    "UserRole": "caseworker-divorce-solicitor",
-    "CRUD": "CRU"
```

Comment:
You need to keep the original in "AuthorisationCaseEvent-prod.json" and then add the version with "R" permission to "AuthorisationCaseEvent-respSol...-nonprod.json"

Reply:
Agreed, Updated here: 3470f36d49a8791a8d9e1b9f7aeac77e49ae212e